ChustaSoft.Tools.Authorization.TestBasic.WebAPI/Startup.cs

```diff
@@ -71,6 +74,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IService
                         ub.Add("ADMIN", "Admn.1234").WithRole("Admin");
                     });
                 ;
+
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllerRoute(
```

Comment:
I think we only have to map the Auth controller, however, we are forcing the client project to use this specific mapping

Reply:
Code removed, as we are adding the AuthorizationController from the Chustasoft.ToolsAuthorization.AspNet assembly in both test projects with the .AddAuthorizationControllers() method